test/e2e/shoot/create_and_delete.go

```diff
@@ -33,6 +45,38 @@ var _ = Describe("Shoot Tests", Label("Shoot"), func() {
 		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
 		f.Verify()
 
+		restConfig := f.GardenerFramework.GardenClient.RESTConfig()
+		versionedClient, err := versioned.NewForConfig(restConfig)
+		Expect(err).NotTo(HaveOccurred())
+
+		adminKubeconfigRequest := &v1alpha1.AdminKubeconfigRequest{
+			Spec: v1alpha1.AdminKubeconfigRequestSpec{
+				ExpirationSeconds: pointer.Int64(3600),
+			},
+		}
+
+		adminKubeconfig, err := versionedClient.CoreV1beta1().Shoots(f.Shoot.GetNamespace()).CreateAdminKubeconfigRequest(ctx, f.Shoot.GetName(), adminKubeconfigRequest, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Create shoot client using adminKubeconfig")
+		Eventually(func(g Gomega) {
+			shootClient, err = kubernetes.NewClientFromBytes(adminKubeconfig.Status.Kubeconfig, kubernetes.WithClientOptions(
+				client.Options{
+					Scheme: kubernetes.ShootScheme,
+				}),
+				kubernetes.WithDisabledCachedClient(),
+			)
+			g.Expect(err).NotTo(HaveOccurred())
+		}).Should(Succeed())
+
+		f.Logger.Infof("ShootClient was created successfully!")
```

Comment:
Do we need these logs? I don't see them being used anywhere else?

Reply:
I added them for debugging purposes and just kept them. I can remove them if you want.

Reply:
IMO they are not needed since we are only interested in failures, and ginkgo will tell us accordingly when a certain assertion does not pass

Reply:
Fair enough.